iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TwoWindingsTransformerImpl.java

```diff
@@ -6,15 +6,11 @@
  */
 package com.powsybl.iidm.network.impl;
 
+import com.powsybl.iidm.network.*;
+
 import java.util.HashSet;
 import java.util.Set;
 
-import com.powsybl.iidm.network.ConnectableType;
-import com.powsybl.iidm.network.Identifiable;
-import com.powsybl.iidm.network.TapChanger;
-import com.powsybl.iidm.network.TwoWindingsTransformer;
-import com.powsybl.iidm.network.ValidationUtil;
```

Comment:
Maybe there is something in your IDE configuration. The import order should remain the same. Maybe we can share our configuration

Reply:
Sorry, my bad
Use Intellij configuration "Optimize imports on the fly"